applications/Unity.GrantManager/src/Unity.GrantManager.Web/appsettings.json

```diff
@@ -39,7 +39,7 @@
     "SecretAccessKey": "",
     "ApplicationS3Folder": "Unity/Application",
     "AssessmentS3Folder": "Unity/Adjudication",
-    "DisallowedFileTypes": "[ \"application/x-msdownload\" ]",
+    "DisallowedFileTypes": "[ \"exe\",\"sh\",\"ksh\",\"bat\",\"cmd\" ]",

```

Comment:
have we tested this in the docker env? (linux, maybe also setting the openshift values in test to test as this was the error last time with these disallowed file types)

Reply:
> have we tested this in the docker env? (linux, maybe also setting the openshift values in test to test as this was the error last time with these disallowed file types)

yup, me and @DarylTodosichuk tested it, the escape character \ should be removed when the value is set in the openshift environment as S3__DisallowedFileTypes.

Reply:
thanks - resolving